projectile.el

```diff
@@ -2640,6 +2640,41 @@ is chosen."
 
 (projectile-commander-bindings)
 
+(defun projectile-skel-read-variable ()
+  "Prompt for a variable and return its name."
+  (completing-read "Variable: "
+                   obarray
+                   '(lambda (v)
+                      (and (boundp v) (not (keywordp v))))
+                   t))
+
+(define-skeleton projectile-skel-variable-cons
```

Comment:
See https://www.gnu.org/software/emacs/manual/html_node/autotype/Skeleton-Language.html
it is a builtin template language.